frontend/pages/index.tsx

```diff
@@ -1,11 +1,6 @@
-import { Center } from '@mantine/core';
+import React from 'react';
+import Homepage from './Homepage';
 
 export default function Home() {
-  return (
-    <>
-      <Center style={{ alignItems: 'center', paddingTop: '80px' }}>
-        <div>Coming soon!</div>
-      </Center>
-    </>
-  );
+  return <Homepage />;
```

Comment:
Probably a matter of opinion, but is it a bit unnecessary to have the homepage be a separate page that gets pulled into the index.tsx? Like homepage is index and index is homepage, so it kind of feels like 2 names for the same thing.

Reply:
Yeah I would just put the contents of the `homepage` component inside `index.tsx` directly.

Reply:
Also putting `Homepage.tsx` in the `pages` directory will create a route to `localhost:3000/Homepage` for the content, which is not what we want. If you were to separate out the content of a page into its own component, use a `components` directory or something. `pages` should be reserved for things that will be directly routed to